app/controllers/application_controller.rb

```diff
@@ -1,11 +1,27 @@
 class ApplicationController < ActionController::Base
```

Comment:
The `current_user_attributes` helper method in `ApplicationController` is exposing all user attributes to the client side. This can lead to sensitive information disclosure. Consider exposing only the necessary attributes.